app/serializers/user_serializer.rb

```diff
@@ -3,6 +3,7 @@
 class UserSerializer < ActiveModel::Serializer
   type "user"
   attributes :sso_guid, :created_at
+  has_many :favorite_tools, key: "favorite-tools"
```

Comment:
this is not being serialized correctly.

What I'm getting:
```
{
    "data": {
        "id": "1",
        "type": "user",
        "relationships": {
            "favorite-tools": {
                "data": [
                    {
                        "id": 1,
                        "tool-id": 1,
                        "user-id": 1,
                        "created-at": "2022-09-16T18:16:15.356Z",
                        "updated-at": "2022-09-16T18:16:15.356Z"
                    },
                    {
                        "id": 2,
                        "tool-id": 2,
                        "user-id": 1,
                        "created-at": "2022-09-16T18:16:15.379Z",
                        "updated-at": "2022-09-16T18:16:15.379Z"
                    },
                    {
                        "id": 3,
                        "tool-id": 10,
                        "user-id": 1,
                        "created-at": "2022-09-16T18:24:57.745Z",
                        "updated-at": "2022-09-16T18:24:57.745Z"
                    }
                ]
            }
        }
    }
}
```

What I'm expecting:
```
{
    "data": {
        "id": "1",
        "type": "user",
        "relationships": {
            "favorite-tools": {
                "data": [
                    {
                        "type": "resource",
                        "id": 1
                    },
                    {
                        "type": "resource",
                        "id": 2
                    },
                    {
                        "type": "resource",
                        "id": 10
                    }
                ]
            }
        }
    }
}
```

Reply:
take note that the `id` being returned should be the id of the tool itself, and not the favorite-tool relationship

Reply:
this is on a `GET /users/me` request

Reply:
this is supposed to represent a many-to-many relationship. Currently you are serializing the intermediate internal db object tracking that many-to-many relationship and not the actual objects on the other side of the relationship.